common/scala/src/main/scala/org/apache/openwhisk/core/connector/Message.scala

```diff
@@ -223,6 +256,15 @@ object Activation extends DefaultJsonProtocol {
       "memory",
       "causedBy")
 
+  /**
+   * Extract namespace and action from name
+   * ex. whisk.system/apimgmt/createApi -> (whisk.system, apimgmt/createApi)
+   */
+  def getNamespaceAndActionName(name: String): (String, String) = {
+    val nameArr = name.split("/", 2)
+    (nameArr(0), nameArr(1))
+  }
+
```

Comment:
Any reason not to split these in the `Activation` definition? (and avoid parsing these later etc)